openeo_udp/variabilitymap/generate.py

```diff
@@ -0,0 +1,108 @@
+"""
+The variability map will divide a polygon into different zones based on the pixel values
+and their relative difference with the mean value of the field.
+A map will be generated for each of the available S2 observation between the specified date range.
+
+"""
+import json
+import sys
+from pathlib import Path
+from typing import Sequence, Union
+
+import openeo
+from openeo.api.process import Parameter
+from openeo.rest.udp import build_process_dict
+from openeo.internal.graph_building import PGNode
+
+
+def load_udf(filename: str) -> PGNode:
+    with open(Path(__file__).parent / filename, 'r') as f:
+        return f.read()
+
+def get_variabilitymap(
+    connection: openeo.Connection,
+    temporal_extent: Union[Sequence[str], Parameter, None] = None,
```

Comment:
bit nitpicking, but `Tuple[str, str]` would be better type annotation:

```suggestion
    temporal_extent: Union[Type[str, str], Parameter, None] = None,
```